decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
```

Comment:
- [ ] You have created the Decorator and Trimmer classes, but you are yet to create the CapitalizeDecorator class, which should inherit from this class, for capitalizing the name. Kindly create it.